app/webpack/shared/components/text_editor.jsx

```diff
@@ -43,6 +52,7 @@ class TextEditor extends React.Component {
                 label={<i className="fa fa-bold" />}
                 template={text => `**${text}**`}
                 placeholder={I18n.t( "bold_text" )}
+                ref={(button) => {this.boldButton = button;}}
```

Comment:
Parens around `button` are extraneous and curly braces around the function definition need spaces on the inside (also in the eslint rules).